src/_data/sidenav.yml

```diff
@@ -51,40 +50,41 @@
           permalink: /docs/development/ui/assets-and-images
         - title: Navigation & routing
           permalink: /docs/development/ui/navigation
-        # Commenting out the following entry per
-        # https://github.com/flutter/website/issues/1496#issuecomment-431455211
         # - title: Inspect an app's UI
         #   permalink: /docs/development/ui/inspector
         - title: Animations
-          permalink: /docs/development/ui/animations
```

Comment:
Ditto: this shouldn't have been removed since it will break breadcrumbs, among other things -- and it likely the cause of #1617.